### DISCUSSION TITLE
How to set eframe always_on_top?

### DISCUSSION THREAD

**Discussion by zhouhang95** [emilk/egui#3898] [Q&A] [answered]
Need help.

**Comment by YgorSouza** [comment]
There is a method [with_always_on_top](https://docs.rs/egui/0.25.0/egui/viewport/struct.ViewportBuilder.html#method.with_always_on_top) in the ViewportBuilder.

**Comment by zhouhang95** [comment]
Could I change this on runtime rather than when creating?

**Reply by YgorSouza** [reply, marked as answer]
I think so. There's a method [send_viewport_cmd](https://docs.rs/egui/0.25.0/egui/struct.Context.html#method.send_viewport_cmd) and one of the variants is [WindowLevel](https://docs.rs/egui/0.25.0/egui/viewport/enum.ViewportCommand.html#variant.WindowLevel).

**Reply by zhouhang95** [reply]
Thanks, it works very well.
